src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSFilter.java

```diff
@@ -42,6 +42,16 @@
     private JavaRDD<GATKRead> reads;
     private final SAMFileHeader header;
 
+    private static final List<String> ADAPTER_SEQUENCES = CollectionUtil.makeList(
```

Comment:
I didn't intend for these sequences to always be used, I just happened to use the same sequences in the test as in PSFilter().